lib/ruby_lsp/test_helper.rb

```diff
@@ -76,5 +77,16 @@ def pop_message(outgoing_queue, &block)
       message = outgoing_queue.pop until block.call(message)
       message
     end
+
+    #: (IO output) -> Array[Hash[Symbol, untyped]]
```

Comment:
Not sure how type checking is not failing, but the return type here is not correct.

The hash keys are strings since we're not using `symbolize_names: true`.